packages/e2e-test-app-fabric/windows/RNTesterApp-Fabric/RNTesterApp-Fabric.cpp

```diff
@@ -62,26 +62,34 @@ void UpdateRootViewSizeToAppWindow(
 winrt::Microsoft::ReactNative::ReactNativeHost CreateReactNativeHost(
     HWND hwnd,
     const winrt::Microsoft::UI::Composition::Compositor &compositor) {
-  WCHAR workingDir[MAX_PATH];
-  GetCurrentDirectory(MAX_PATH, workingDir);
+  WCHAR appDirectory[MAX_PATH];
+  GetModuleFileNameW(NULL, appDirectory, MAX_PATH);
+  PathCchRemoveFileSpec(appDirectory, MAX_PATH);
 
   auto host = winrt::Microsoft::ReactNative::ReactNativeHost();
-  // Disable until we have a 3rd party story for custom components
-  // RegisterAutolinkedNativeModulePackages(host.PackageProviders()); // Includes any
-  // autolinked modules
+
+  // Include any autolinked modules
+  //RegisterAutolinkedNativeModulePackages(host.PackageProviders());
+
   host.PackageProviders().Append(winrt::make<RNTesterAppReactPackageProvider>());
   host.PackageProviders().Append(winrt::AutomationChannel::ReactPackageProvider());
 
+#if BUNDLE
   host.InstanceSettings().JavaScriptBundleFile(L"index.windows");
-  host.InstanceSettings().DebugBundlePath(L"index");
```

Comment:
Is this not needed?

Reply:
It doesn't appear to be needed anymore, since now we set JavaScriptBundleFile to index in debug (like we did for UWP)